src/main/java/BlueTurtle/TSE/JSONFormatter.java

```diff
@@ -36,17 +36,17 @@ public void format() throws IOException {
 		List<Warning> totalWarnings = new ArrayList<Warning>();
 		
 		File file = new File(JavaController.getCheckStyleOutputFile());
-		if(file.exists() && !file.isDirectory()) { 
+		if(file.exists()) { 
```

Comment:
Btw I noticed that is it not necessary to pass the parser to the function, because the parser is a attribute of `JSONFormatter`.